tools/mason/MasonExternal.chpl

```diff
@@ -18,7 +18,9 @@
  * limitations under the License.
  */
 
-config const spackVersion = "releases/v0.11.2";
+config const version = '0.14.2';
+config const branch = 'releases/v0.14';
```

Comment:
It'd be ideal to construct this string from `spackVersion` if that's safe to do, i.e. prepend `releases/v` and remove the last 2 characters. The advantage being that it is one less thing to update each time we update.